misc/glibc/vars.yaml

```diff
@@ -1,4 +1,4 @@
 # renovate: datasource=docker versioning=docker depName=cgr.dev/chainguard/wolfi-base
-WOLFI_BASE_REF: sha256:72c8bfed3266b2780243b144dc5151150015baf5a739edbbde53d154574f1607
+WOLFI_BASE_REF: sha256:426720c517a540425dc85e046ed9503f1980c989889d2062b70cb1b0b39fa7d8
```

Comment:
this is the amd64 digest, not the combined digest that supports both amd64 and arm64

```
❯ crane digest cgr.dev/chainguard/wolfi-base
sha256:0c1bf69476e3ca3d4763ca3067773e8796a1faecd56678a3b748cd90cfb9b9a5
```

should be using above

```
❯  crane --platform=linux/amd64 digest cgr.dev/chainguard/wolfi-base
sha256:426720c517a540425dc85e046ed9503f1980c989889d2062b70cb1b0b39fa7d8
```

Reply:
Oops, will update.